rotkehlchen/db/dbhandler.py

```diff
@@ -181,6 +183,40 @@
 # http://www.sql-join.com/sql-join-types
 
 
+def insert_into_credentials_mappings(
```

Comment:
you can reuse this for other credential mappings and add it to the `DBHandler` class. should have accept a dict for the key-value entries.